src/index.ts

```diff
@@ -1,5 +1,48 @@
+import {EvernoteConvertor} from './convertors/evernote'
+import {HtmlConvertor} from './convertors/html'
+import {LogseqConvertor} from './convertors/logseq'
+import {MarkdownConvertor} from './convertors/markdown'
+import {MemConvertor} from './convertors/mem/mem-convertor'
+import {OpmlConvertor} from './convertors/opml'
+import {RoamConvertor} from './convertors/roam'
+
 export * from './types'
 export * from './convertor'
 export * from './convertors/evernote'
 export * from './convertors/markdown'
 export * from './convertors/roam'
+
+export type Format = 'evernote' | 'html' | 'logseq' | 'markdown' | 'mem' | 'opml' | 'roam'
+
+export const convertors = {
+  evernote: EvernoteConvertor,
+  html: HtmlConvertor,
+  logseq: LogseqConvertor,
+  markdown: MarkdownConvertor,
+  mem: MemConvertor,
+  opml: OpmlConvertor,
+  roam: RoamConvertor,
+}
+
+// Returns something like:
+// {
+//   "evernote": [
+//     {
+//       "description": "Evernote ENEX",
+//       "accept": {
+//         "application/enex": [
+//           ".enex"
+//         ]
+//       }
+//     }
+//   ],
+export const convertorTypes: Record<Format, FilePickerAcceptType[]> = Object.entries(
+  convertors,
+).reduce((acc, [key, convertor]) => {
+  const instance = new convertor({graphId: 'test'})
```

Comment:
If we set `accept` and `description` as static, then I guess we don't need to create a "fake" convertor instance to get these two values.

```
export abstract class Convertor {
   static accept: ConvertorAcceptType
   static description: string
```

Reply:
Yeah, I would much rather that. Unfortunately, it doesn't look like you can have... abstract statics.

Reply:
I see your point. You cannot force the child class to implement static attributes